src/base/bootstrap-overrides/_base.scss

```diff
@@ -45,8 +45,24 @@ a {
 }
 
 .dl-horizontal {
-	dt {
-		white-space: normal;
+	@media (min-width: 768px) {
```

Comment:
Maybe so, but it is also defined in a variable so shouldn't we stick to the variables? If Bootstrap changes from 768px to 769px then they are responsible for updating all their variables and code but by us using a variable then we wouldn't have to make a change if that scenario were to happen.